modules/machineset-gcp-confidential-vm.adoc

```diff
@@ -13,10 +13,12 @@ endif::[]
 
 By editing the machine set YAML file, you can configure the Confidential VM options that a machine set uses for machines that it deploys.
 
-For more information about Confidential Compute features, functionality, and compatibility, see the GCP Compute Engine documentation about link:https://cloud.google.com/compute/confidential-vm/docs/about-cvm[Confidential VM].
+For more information about Confidential Compute features, functions, and compatibility, see the GCP Compute Engine documentation about link:https://cloud.google.com/compute/confidential-vm/docs/about-cvm[Confidential VM].
 
-:FeatureName: Confidential Computing
-include::snippets/technology-preview.adoc[]
+[IMPORTANT]
+====
+{product-title} {product-version} does not support some Confidential Compute features, such as confidential VMs with AMD Secure Encrypted Virtualization Secure Nested Paging (SEV-SNP).
```

Comment:
"Confidential VM" is regularly capitalized in the documentation that we're linking to. Should we capitalize it here as well?

Reply:
Yes, ty!